frontend/src/pages/UpdateUser.tsx

```diff
@@ -46,11 +46,11 @@ export const UpdateUser = () => {
     try {
       const token = getToken();
```

Comment:
This shouldn't be in a tsx file. Let's move it into the `updateUser` function in the api folder

Reply:
Just fixed that up. Will let you have final look to make sure it's all good and then will merge it in.